### PR TITLE
feat(artist): exposes collected artworks count

### PR DESCRIPTION
**Pull request by dzucconi**
The count in the collection table should be the count of the artworks you've collected.

### CODE REVIEW

File: src/schema/v2/artist/index.ts [hunk 1]
```diff
@@ -627,6 +627,24 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
...
               },
             },
+            myCollectedArtworks: {
+              type: new GraphQLNonNull(GraphQLInt),
+              resolve: async ({ _id }, _, { meMyCollectionArtworksLoader }) => {
+                if (!meMyCollectionArtworksLoader) {
+                  return 0
+                }
+
+                const { headers } = await meMyCollectionArtworksLoader({
```
Comment (araujobarret) [inline src/schema/v2/artist/index.ts:637]: how about making it more error-proof if the request fails?

Comment (dzucconi) [reply src/schema/v2/artist/index.ts:637]: In what way: you wouldn't want to return an erroneous count (0). You want it to error and return the error.

Comment (araujobarret) [reply src/schema/v2/artist/index.ts:637]: true but the client consuming it will have the entire query request broken, isn't it?

Comment (dzucconi) [reply src/schema/v2/artist/index.ts:637]: Depends on how one handles the error. An error in the field would null the parent like so: 


Either way: IMO you don't want to swallow the error and return null on the field. You want to surface the error for the client to deal with.

I'd rather the page error than have to include error handling code on every field I display on the front end.

Comment (araujobarret) [reply src/schema/v2/artist/index.ts:637]: agreed, handling the errors in the client sounds more real IMO than fake nulls.

**Review by araujobarret — APPROVED** [review]

**Pull request merged**